src/app/component/eco-news/components/create-news/create-news.component.scss

```diff
@@ -324,4 +327,8 @@ form label {
     display: flex;
     flex-direction: column;
   }
+
+  .date p:last-child {
+    margin: 0 0 43px;
```

Comment:
Why 43 px?

Reply:
According to mock-up
![2020-09-07](https://user-images.githubusercontent.com/62401277/92397496-92fc8100-f12f-11ea-9ae6-894e522e0118.png)